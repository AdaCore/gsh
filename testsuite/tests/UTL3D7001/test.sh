counter=0
. sourced_script.sh
echo "counter: $counter"
mkdir subdir
cd subdir

. ../sourced_script.sh
echo "counter: $counter"
