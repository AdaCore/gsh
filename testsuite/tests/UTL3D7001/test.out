counter: 1
counter: 2
